daemons/fenced/fenced_commands.c

```diff
@@ -139,18 +139,38 @@ get_action_delay_max(stonith_device_t * device, const char * action)
 }
 
 static int
-get_action_delay_base(stonith_device_t * device, const char * action)
+get_action_delay_base(stonith_device_t *device, const char *action, const char *victim)
 {
-    const char *value = NULL;
+    char *hash_value = NULL;
     int delay_base = 0;
 
     if (!pcmk__strcase_any_of(action, "off", "reboot", NULL)) {
         return 0;
     }
 
-    value = g_hash_table_lookup(device->params, PCMK_STONITH_DELAY_BASE);
-    if (value) {
-       delay_base = crm_parse_interval_spec(value) / 1000;
+    hash_value = g_hash_table_lookup(device->params, PCMK_STONITH_DELAY_BASE);
+
+    if (hash_value) {
+        char *value = strdup(hash_value);
+        char *valptr = value;
+
+        if (victim) {
+            for (char *val = strtok(value, "; \t"); val != NULL; val = strtok(NULL, "; \t")) {
+                char *mapval = strchr(val, ':');
+
```

Comment:
In case someone misconfigures it, we should log an error if mapval is NULL or val, or mapval[1] = 0

We can continue the loop though